lib/ansible/modules/tacp/tacp_info.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2020, Lenovo
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.tacp_ansible import tacp_utils
+
+
+import json
+import tacp
+import sys
+from tacp.rest import ApiException
+from pprint import pprint
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: tacp_info
+
+short_description: This is my test module
+
+version_added: "2.9"
+
+description:
+    - "This is my longer description explaining my test module"
+
+options:
+    name:
+        description:
+            - This is the message to send to the test module
+        required: true
+    new:
+        description:
+            - Control to demo if the result of this module is changed or not
+        required: false
+
+extends_documentation_fragment:
+    - tacp
+
+author:
+    - Xander Madsen (@xmadsen)
+'''
+
+EXAMPLES = '''
+# Pass in a message
+- name: Test with a message
+  tacp_instance:
+    name: hello world
+
+# pass in a message and have changed true
+- name: Test with a message and changed output
+  tacp_instance:
+    name: hello world
+    new: true
+
+# fail the module
```

Comment:
I don't understand this. What is failing here?

Reply:
These documentation bits will be populated in another story, they'll provide actual examples of usage